trustly-android-sdk/publish-remote.gradle

```diff
@@ -54,9 +54,9 @@ afterEvaluate {
                     }
                     developers {
                         developer {
-                            id = 'andre.guedes@trustly'
-                            name = 'Andre Guedes'
-                            email = 'andre.guedes@trustly.com'
+                            id = 'ops-root@paywithmybank.com'
+                            name = 'TrustlyInc'
+                            email = 'ops-root@paywithmybank.com'
```

Comment:
Shouldn't this be the devX alias? Or is the ops team the right contanct? (amer.developer.experience@trustly.com)

Reply:
This is the DevOps team contact, @pgonring added this info when we publish the first version to the Maven Central